roles/foreman_installer/tasks/install.yml

```diff
@@ -21,6 +21,11 @@
     foreman_installer_options_joined: "{{ foreman_installer_options_joined | replace('--certs-tar-file', '--foreman-proxy-content-certs-tar') }}"
   when: foreman_installer_version_file['content'] | b64decode is version('1.21', '<')
 
+- name: 'Replace --inital-admin parms with --admin legacy params'
+  set_fact:
+    foreman_installer_options_joined: "{{ foreman_installer_options_joined | replace('--foreman-admin-password', '--foreman-admin-username', '--foreman-admin-email', '--foreman-admin-first-name', '--foreman-admin-last-name') }}"
```

Comment:
This does not what you think it does. In the previous example it replaces `--certs-tar-file` with `--foreman-proxy-content-certs-tar`. You only specify the old options.